cpp/oneapi/dal/algo/svm/backend/cpu/infer_kernel.cpp

```diff
@@ -49,19 +49,19 @@ static infer_result call_daal_kernel(const context_cpu& ctx,
     auto arr_data = row_accessor<const Float>{ data }.pull();
     auto arr_support_vectors =
         row_accessor<const Float>{ trained_model.get_support_vectors() }.pull();
-    auto arr_coefficients = row_accessor<const Float>{ trained_model.get_coefficients() }.pull();
+    auto arr_coeffs = row_accessor<const Float>{ trained_model.get_coeffs() }.pull();
```

Comment:
Can we introduce some simple rule to shorten the names? We have `get_coeffs()` in the model but `get_column_count()` in tables, which can be shorten also to `get_col_count()`

Reply:
We have to negotiate within the library. Coefficients have been cut due to the fact that the graphs have a approved name `coeff`. (https://github.com/oneapi-src/oneDAL/pull/823/files#diff-38367a019750050cdad24bcdde40989aR94)